library/logger.hpp

```diff
@@ -99,7 +100,14 @@ template <class T> inline blackhole::logger_facade<dnet_logger> make_facade(T &&
 
 #else
 #define DNET_LOG_RAW(...)		dnet_log_raw(__VA_ARGS__)
-#define DNET_LOG(node, ...)		DNET_LOG_RAW(dnet_node_get_logger(node), __VA_ARGS__)
+
+#define DNET_LOG(__node__, __severity__, ...)				\
+	do {								\
+		if (dnet_node_logging_allowed(__node__, __severity__)) { \
```

Comment:
Please, use `int dnet_node_get_verbosity(struct dnet_node *n)` and also check `trace_bit` here.